CHANGELOG.md

```diff
@@ -15,8 +15,38 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+### Changed
+- *ORCA-290* Renamed `excludeFileTypes` to `orcaExcludedFileExtensions`. In addition, ORCA configuration variables `orcaExcludedFileExtensions`, `orcaDefaultBucketOverride` and `orcaDefaultStorageClassOverride` are now under a new key `orca` under `meta` key.  
+
+### Migration Notes
+
+- Adjust workflows/step functions for `OrcaCopyToGlacierWorkflow`.
+  - `excludeFileTypes` argument in `task_config` is now `orcaExcludedFileExtensions`.
+  - `orcaExcludedFileExtensions`, `orcaDefaultBucketOverride` and `orcaDefaultStorageClassOverride` keys are now under a new key `orca`. See example below:
+```json
+"task_config": {
+  "orcaExcludedFileExtensions": "{$.meta.collection.meta.orca.orcaExcludedFileExtensions}",
+  "orcaDefaultBucketOverride": "{$.meta.collection.meta.orca.orcaDefaultBucketOverride}",
+  "orcaDefaultStorageClassOverride": "{$.meta.collection.meta.orca.orcaDefaultStorageClassOverride}"
+}
+```
+```json
+"collection": {
+    "meta":{
+        "orca": {
+          "orcaDefaultStorageClassOverride": "DEEP_ARCHIVE",
+          "orcaExcludedFileExtensions": [".xml"],
+          "orcaDefaultBucketOverride": "orca-bucket"
+      }
+  }
+}
+
+```
+- Adjust workflows/step functions for `OrcaRecoveryWorkflow`.
```

Comment:
Move the information for the OrcaRecoveryWorkflow under changed as this is not something an end user will need to do. This is something we own.